From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: remove runtime test for cross compiling

When cross compiling, we assume that the user has made sure that
reasonably new versions are provided.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 configure.in |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

Index: mod_python-3.3.1/configure.in
===================================================================
--- mod_python-3.3.1.orig/configure.in
+++ mod_python-3.3.1/configure.in
@@ -108,6 +108,7 @@ else
 
   # check Apache version
   AC_MSG_CHECKING(Apache version)
+  if test "x$cross_compiling" = "xno"; then
   HTTPD="`${APXS} -q SBINDIR`/`${APXS} -q TARGET`"
   ver=`$HTTPD -v | awk '/version/ {print $3}' | awk -F/ '{print $2}'`
   AC_MSG_RESULT($ver)
@@ -126,7 +127,9 @@ else
   AC_MSG_CHECKING([for Apache include directory])
   AP_INCLUDES="-I`${APXS} -q INCLUDEDIR`"
   AC_MSG_RESULT($AP_INCLUDES)
-
+  else
+    AC_MSG_RESULT([cross_compiling, assuming recent new version...])
+  fi
   dnl Small hack to work around _eprintf.o problem on Solaris
   if test "`uname`" = "SunOS"; then
     AC_MSG_CHECKING([for gcc on Solaris possible missing _eprintf problem])
